docs/tutorials/platform_beginner/4_display_a_list_of_content_items.md

```diff
@@ -82,8 +101,65 @@ class HomepageController extends Controller
 }
 ```
 
-This controller searches for all visible Content items of the type **Ride** (lines 39-40)
-and renders them using the `list/rides.html.twig` template (line 28).
+This controller searches for all visible Content items of the type **Ride** (lines 58-59)
+and renders them using the `list/rides.html.twig` template (line 45).
+
+### Register a service
+
+!!! tip
+
+    Dealing with Symfony services is not in the scope of this tutorial, so we won't go here into detail on how they work.
+    To learn more about this topic, take a look at [Symfony Service Container documentation](https://symfony.com/doc/3.4/service_container.html).
+
+Because you are using the `ContentService`, you need to register the `HomepageController` as a service.
+
+Create a new file: `src/AppBundle/Resources/config/services.yml`:
+
+``` yaml
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    AppBundle\Controller\:
+        public: true
+        resource: '../../Controller/*'
+```
+
+At the beginning of the `ezplatform.yml` file (before the `ezpublish` key), add the following lines to import the content of the new file:
+
+``` yaml
+imports:
+    - { resource: '@AppBundle/Resources/config/services.yml' }
+```
+
+You also need to use dependency injection to merge your service into the actual container.
+
+Create a new file `src/AppBundle/DependencyInjection/AppExtension.php`:
```

Comment:
Don't think we need this at basic level.

Also Symfony lists services in app/config so if we end up needing to define services we can do that too:
- 3.4: https://github.com/symfony/symfony-standard/blob/3.4/app/config/services.yml
- 4.x: https://github.com/symfony/demo/blob/master/config/services.yaml